docs/populate.md

```diff
@@ -804,6 +805,31 @@ MySchema.post('save', function(doc, next) {
 });
 ```
 
+<h3 id="populating-multiple-paths-middleware"><a href="#populating-multiple-paths-middleware">Populating Multiple Paths in Middleware</a></h3>
+
+Populating multiple paths in middleware is just a little bit trickier than what you may think. Here's how you may expect it to work:
+
+```javascript
+userSchema.pre('find', function (next) {
```

Comment:
Would be great to add the schema definition to this example: it isn't clear that `followers` and `following` reference `User`

Reply:
Agreed, will do :)